e2e/tests/recycled_storage.json

```diff
@@ -5,18 +5,18 @@
       "origin": "http://localhost:8080",
       "localStorage": [
         {
-          "name": "tcHistory",
-          "value": "{\"utc\":[{\"start\":1652301954635,\"end\":1652303754635}]}"
+          "name": "mct-tree-expanded",
```

Comment:
Should this file have been changed or it doesn't matter?

Reply:
@ozyx unfortunately firefox and chrome switching the order of the objects in session storage. We will need to address as a part of https://github.com/nasa/openmct/issues/5330